### PR TITLE
zypper: add pt_BR translation

### PR DESCRIPTION
**Pull request by isaacvicente**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by isaacvicente** [comment]
In order to follow the style guide page of tldr (imperative mood section), can a maintainer make the following changes?
- "Sincrozine" instead of "Sincronizar"
- "Instale" instead of "Instalar"
- "Remova" instead of "Remover"
- "Atualize" instead of "Atualizar"
- "Pesquise um" instead of "Pesquisar"
- "Mostre" instead of "Mostrar"

**Review by kbdharun — APPROVED** [review]
LGTM after implementing the above changes, Thanks for your contribution.

Edit: I will make the changes.

File: pages.pt_BR/linux/zypper.md
Comment (navarroaxel) [inline pages.pt_BR/linux/zypper.md:6]:
```suggestion
- Sincroniza a lista de pacotes e versões disponíveis:
```

File: pages.pt_BR/linux/zypper.md
Comment (navarroaxel) [inline pages.pt_BR/linux/zypper.md:10]:
```suggestion
- Instala um novo pacote:
```

File: pages.pt_BR/linux/zypper.md
Comment (navarroaxel) [inline pages.pt_BR/linux/zypper.md:14]:
```suggestion
- Remove um pacote:
```

File: pages.pt_BR/linux/zypper.md
Comment (navarroaxel) [inline pages.pt_BR/linux/zypper.md:18]:
```suggestion
- Atualiza os pacotes instalados para as versões mais recentes disponíveis:
```

File: pages.pt_BR/linux/zypper.md
Comment (navarroaxel) [inline pages.pt_BR/linux/zypper.md:22]:
```suggestion
- Pesquisa pacote por palavra-chave:
```

File: pages.pt_BR/linux/zypper.md
Comment (navarroaxel) [inline pages.pt_BR/linux/zypper.md:26]:
```suggestion
- Mostra informações relacionadas aos repositórios configurados:
```

**Review by navarroaxel — COMMENTED** [review]
Remember that for translations you can send more than 1 translated page in 1 PR, just keep 1 commit per file and I recommend no more than 15 pages. This doesn't apply to the main «English» pages.

**Comment by isaacvicente** [comment]
Thank you guys for the help, I appreciate it.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**
